bokeh/core/_templates/autoload_nb_js.js

```diff
@@ -1,5 +1,102 @@
 {% extends "autoload_js.js" %}
 
+{% block register_mimetype %}
+
+  var MIME_TYPE = 'application/vnd.bokehjs_exec.v0+json';
+  var CLASS_NAME = 'output_bokeh rendered_html';
+
+  /**
+   * Render data to the DOM node
+   */
+  function render(props, node) {
+    var div = document.createElement("div")
+    div.innerHTML = props["data"]["div"]
+    var script = document.createElement("script")
+    script.textContent = props["data"]["script"]
```

Comment:
Semicolon insertion is a tricky subject, so I would recommend being consistent with semicolons in JS. Either not use them at all, unless necessary (knowing the rules), or play it safe and using them everywhere. My approach is to not use semicolons in typescript (due to compiler's feedback) and use then in plain javascript.